### ISSUE TITLE
macOS accents popup doesn't have keyboard focus

### ISSUE THREAD

**Issue by l65l** [microsoft/vscode#93294]
Issue Type: <b>Bug</b>

When a user long press a keyboard key to make the accent popup appears, the popup shows up but the popup doesn't have keyboard focus. If the user press arrows to navigate the different accent options, the cursor moves in the editor window so the accentuated character is not applied at the good position.

VS Code version: Code 1.43.1 (fe22a9645b44368865c0ba92e2fb881ff1afce94, 2020-03-18T07:01:41.240Z)
OS version: Darwin x64 19.0.0

**Comment by alexdima** [comment]
Duplicate of #67739 . The fix for #67739 is available in the Insiders Channel.
